app/posts/2021-10-23-end-to-end-application-journey-round-3.md

```diff
@@ -0,0 +1,295 @@
+
+6. We tested a feature where hiring staff were able to either download all of the applications at once or share the applications directly to an email address.
+
+{{ appFigure({
+  image: {
+    file: "share-1.png",
+    alt: "initial step of the share flow"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "share-4.png",
+    alt: "success banner once applications have been shared"
+  }
+}) }}
+
+We found that the role of administrative staff is purely to pass all of the applications on to the panel. More senior hiring staff (HR managers of business managers for example) may download and review the applications before passing them on. Having the option to both download or share all of the applications simplified this process.
+
+This also highlighted the varying levels of responsibility amongst hiring staff and potential need for more tailored user experiences.
+
+7. Schools are heavily reliant on having hard copies of applications to assist them with the shortlisting process. This is mainly for speed, as applications are compared side by side and notes are made directly onto the paper applications. Hard copies are stored in files, which they seem reluctant to discard.
+
+We found our current download does not fully support the needs of the SLT as it’s format is not optimal for comparing and quickly reviewing a candidate's information.
+
+8. We tested a feature where hiring staff were able to make notes within an application. These notes would be seen by other members of the recruitment team.
+
+{{ appFigure({
+  image: {
+    file: "notes-1.png",
+    alt: "the note tab, within the application screen"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "notes-3.png",
+    alt: "initial step of the add a note flow"
+  }
+}) }}
+
+We found notes are made by both hiring staff and the SLT both before and after shortlisting has taken place. Sticky notes, handwritten notes and  highlighted text communicate red flags such as issues around references, right to work in the UK, if that candidate had appeared rude during any contact that had been made or to mark areas of the application that seemed incomplete.
+
+Hiring staff add these notes directly onto the applications before they share them with the panel. We found it is unlikely the panel would see the notes made on TV (pre-shortlisting) unless they were actually part of the download.
+
+9. We tested a feature where hiring staff are able to communicate with an applicant if their application has been unsuccessful.
+
+{{ appFigure({
+  image: {
+    file: "reject-1.png",
+    alt: "initial step of the reject application flow"
+  }
+}) }}
+
+{{ appFigure({
+  image: {
+    file: "reject-1.png",
+    alt: "second step of the reject application flow"
+  }
+}) }}
+
+We found HS don't like the term reject, they feel hesitant to use it as it seems both ‘harsh’ and too 'final.’
```

Comment:
I'd expand the HS acronym. Also do you mean hiring staff here, or a subset of hiring staff?